test/framework/src/test/java/org/elasticsearch/test/VersionUtilsTests.java

```diff
@@ -195,7 +199,12 @@ public void testGradleVersionsMatchVersionUtils() {
                 /* Gradle skips the current version because being backwards compatible
                  * with yourself is implied. Java lists the version because it is useful. */
                 .filter(v -> v != Version.CURRENT)
-                .map(v -> v.major + "." + v.minor + "." + v.revision)
+                /* Note that gradle does *not* skip alphas, betas, or rcs here even
+                 * though we don't have backwards compatibility for alphas, betas,
```

Comment:
I thought we agreed to only have the last beta/rc in the wire/bwc test in the special case of a new major release like we have now. So we should not have betas/rcs here in most cases. Also, gradle still does not ever have alphas, that would not be matched by the regex for Version.java.